app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
```

Comment:
- [Optional] There's no reason to explicitly point the various tables since we didn't alter the user for both comments and likes table as we did for posts table. Rails by default can easily figure it out :+1:

Reply:
remove the explicitly point in user models and post models